README.md

```diff
@@ -1,6 +1,29 @@
-# Portfolio
+# Rhet Pulley - SQL Developer, Data Analyst, Report Writer, ETL Developer
+<img src="https://user-images.githubusercontent.com/108885804/210109116-8a16571a-eb1b-4ac6-941b-74c8c413797c.jpg" height="100px"/>  Rwpull@gmail.com  
+
+
+For the past year I have been able to learn Microsoft SQL, PostgreSQL and MySQL through training with a Senior SQL Developer. 
+I have experience and training in multiple different areas including:
+|SQL Experience|GitHub Experience|                                                                    
+|---|---|                                                                                
+|Common SQL data types and database normalization|Basic Git action such as fetch, pull, push and commit|                                  
+|UNIONs, JOINs, subqueries, case statements|Branching, creating pull requests, merging pull requests, handling merge issues|                                       
+|Transactions, catching and handling errors, rolling back changes|Query optimization and execution plans|                 
+|Higher functions such as Common Table Expressions, Windows functions, PIVOTs|<p align="center">  **ETL Experience** </p>  |  
+|Stored procedures, User Defined Functions, Table Valued Functions|Basic principles of Extract, Translate and Load|  
+|Views, masking PPI, encryption, storing sensitive data|Some experience with SSIS and Microsoft Azure|  
+|Query optimization and execution plans|Created Stored Procedures to ETL a spreadsheet| 
+
+I also know how to use Microsoft Report Builder and PowerBI in order to visual business strategies in my previous business. 
```

Comment:
Please break these up with headers.  That's kind of how it's done in Github.  So the first section would be ## SQL experience, next section would be ## Git Experience, next section would be ## Data Analytics Experience